src/main/java/net/fabricmc/loom/LoomGradleExtension.java

```diff
@@ -355,7 +380,7 @@ public boolean ideSync() {
 	// Ideally this should use maven, but this is a lot easier
 	public Function<String, String> getIntermediaryUrl() {
 		// Done like this to work around this possibly not being a java string...
-		return s -> intermediaryUrl.apply(s).toString();
+		return s -> intermediaryUrl.apply(UrlEscapers.urlFragmentEscaper().escape(s)).toString();
```

Comment:
No, I wouldnt do this here? why this unrelated change?

Reply:
Related since `MojmapCreator` uses this to download the intermediaries, and it would need to escape the version in both `MappingsProvider` and `MojmapCreator` anyway